staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -526,6 +529,25 @@ const (
 	PersistentVolumeClaimFileSystemResizePending PersistentVolumeClaimConditionType = "FileSystemResizePending"
 )
 
+type PersistentVolumeClaimResizeStatus string
```

Comment:
This needs a comment and should probably be flagged `+enum` (@jiahuif FYI) as in https://github.com/kubernetes/kubernetes/pull/105057.

Reply:
I have added `+enum` here. I wonder if I should have waited for other feature, we will see.